content/i18n/en/landing-page.json

```
@@ -27,32 +28,31 @@
   "keyFeatures": {
     "headline": "Key features",
     "paragraph": "Kyma {spelling} is a platform for extending applications with serverless functions and microservices.",
-    "spelling": "/kee-ma/",
     "features": {
       "extensibility": {
-        "title": "Extensibility platform",
+        "title": "Serverless",
         "paragraphs": [
-          "If you are looking for cloud-native ways to extend your enterprise products, Kyma is a secure and tested environment you can rely on.",
-          "First, connect external APIs and ensure a stable event flow between the external or built-in services and your own applications. Then, start molding your own code into microservices and functions that extend your applications' logic."
+          "Kyma empowers developers to easily deploy workloads in form of functions. With Kyma you can focus on coding.",
+          "Simply write your function code, add a configuration manifest and push it to Kyma using supporting CLI or directly to git."
```

```suggestion
          "Simply write your Function code, add a configuration manifest, and push it to Kyma using supporting CLI or directly to git."
```
For "supporting CLI", I think we need an article (_a_ or _the_), or if we refer to Kyma CLI specifically, write something like "using _our_ supporting CLI", or simply "using Kyma CLI".

---

Added `our` to make it sound better. thx